app/views/partials/layout/_navbar.html.erb

```diff
@@ -21,6 +21,13 @@
         </ul>
 
         <div class="container text-center d-flex justify-content-end gap-4">
+          <div class="row">
+            <% if user_signed_in? %>
+              <%= link_to 'SIGN OUT', destroy_user_session_path, class: 'btn btn-secondary', data: { turbo_method: :delete } %>
+            <% else %>
+              <%= link_to "LOGIN" , user_session_path, class: 'btn btn-dark' %>
+            <% end %>
```

Comment:
try to ever use unique kinds of quotes if is `""` use them in all your code, and if it is `''` used it in the same way.